### DISCUSSION TITLE
Documentation on how to include a CodeBlock in a Tab

### DISCUSSION THREAD

**Discussion by lanegoolsby** [facebook/docusaurus#8106] [Ideas]
Is [this](https://github.com/apify/crawlee/blob/2a2040e13209aff5e64ee47194940182b686b3a7/website/versioned_docs/version-3.0/introduction/03-adding-urls.mdx?plain=1#L9-L16) a correct approach for adding a CodeBlock in a tab?

I only ask because according to the README for [raw-loader](https://github.com/webpack-contrib/raw-loader) the project is archived and advocates for Asset Modules instead.

I plan on submitting a PR with guidance on how to add code blocks into a Tab. What the link above is doing is exactly what we need to do and the docs are sorely lacking in this area. 

Before I spend the time writing I just want to make sure I'm barking up the right tree. I'm not familiar enough with DS and its toolchain (esp. Webpack) and overall vision to know if I should target Asset Modules instead of using an obsolete library in my examples. 
 
But having a write-up of something akin to that would be a massive help for a lot of folks.

**Comment by slorber** [comment]
Documented here: https://docusaurus.io/docs/markdown-features/react#importing-code-snippets

(probably not in the best location)

> I only ask because according to the README for [raw-loader](https://github.com/webpack-contrib/raw-loader) the project is archived and advocates for Asset Modules instead.

Yes we want to move away from loaders in favor of assets. See also 
https://github.com/facebook/docusaurus/pull/4708
https://github.com/facebook/docusaurus/issues/4445#issuecomment-1249276192
